signer/xpi/README.md

```diff
@@ -73,14 +72,6 @@ signers:
         partner: true
       relative_start: 0h
       duration: 26298h
-    # RSA key gen is slow and CPU intensive, so we can optionally
-    # pregenerate and cache keys with a worker pool
-    rsacacheconfig:
-      numkeys: 25
-      numgenerators: 2
-      generatorsleepduration: 1m
-      fetchtimeout: 100ms
-      statssamplerate: 1m
```

Comment:
AMO has the same config for localdev, see: https://github.com/mozilla/addons-server/blob/78a854871370bb2d53940a70b4048dc203be4a30/scripts/autograph_localdev_config.yaml#L142-L149 -  I am guessing that change will either ignore or break the localdev Autograph service? In any case, I'll file an issue to update the config.

Reply:
Yeah, I'm not sure if it'll break or not, but there's obviously no reason to remove it soon!

Reply:
For clarity: the options will, yep, definitely be ignored in the yaml file once you update to this version of autograph and you won't see a break.